roles/openshift_examples/defaults/main.yml

```diff
@@ -6,7 +6,9 @@ openshift_examples_load_db_templates: true
 openshift_examples_load_xpaas: "{{ openshift_deployment_type in ['enterprise','openshift-enterprise','atomic-enterprise','online']  }}"
 openshift_examples_load_quickstarts: true
 
-examples_base: /usr/share/openshift/examples
+content_version: "{{ 'v1.0' if not openshift.common.version_greater_than_3_1_or_1_1 else 'v1.1' }}"
```

Comment:
I think that switching this to: `"{{ 'v1.1' if openshift.common.version_greater_than_3_1_or_1_1 else 'v1.0' }}"` is a bit clearer.